code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -47,394 +47,665 @@
-
-	feather
-		name = "Feather"
-		icon_state = "feather"
-
-	hitop
-		name = "Hitop"
-		icon_state = "hitop"
-		gender = MALE
-
-	mohawk
-		name = "Mohawk"
-		icon_state = "d"
+/datum/sprite_accessory/hair/short
+	name = "Short Hair"	  // try to capatilize the names please~
+	icon_state = "a" // you do not need to define _s or _l sub-states, game automatically does this for you
 
-	jensen
-		name = "Adam Jensen Hair"
-		icon_state = "jensen"
-		gender = MALE
-
-	gelled
-		name = "Gelled Back"
-		icon_state = "gelled"
-		gender = FEMALE
-
-	gentle
-		name = "Gentle"
-		icon_state = "gentle"
-		gender = FEMALE
-
-	spiky
-		name = "Spiky"
-		icon_state = "spikey"
-
-	kusangi
-		name = "Kusanagi Hair"
-		icon_state = "kusanagi"
-
-	kagami
-		name = "Pigtails"
-		icon_state = "kagami"
-		gender = FEMALE
-
-	himecut
-		name = "Hime Cut"
-		icon_state = "himecut"
-		gender = FEMALE
-
-	braid
-		name = "Floorlength Braid"
-		icon_state = "braid"
-		gender = FEMALE
-
-	mbraid
-		name = "Medium Braid"
-		icon_state = "shortbraid"
-		gender = FEMALE
-
-	braid2
-		name = "Long Braid"
-		icon_state = "hbraid"
-		gender = FEMALE
-
-	braid3
-		name = "Long Braid 2"
-		icon_state = "longbraid"
-		gender = FEMALE
-
-	odango
-		name = "Odango"
-		icon_state = "odango"
-		gender = FEMALE
-
-	ombre
-		name = "Ombre"
-		icon_state = "ombre"
-		gender = FEMALE
-
-	updo
-		name = "Updo"
-		icon_state = "updo"
-		gender = FEMALE
-
-	skinhead
-		name = "Skinhead"
-		icon_state = "skinhead"
-
-	balding
-		name = "Balding Hair"
-		icon_state = "e"
-		gender = MALE // turnoff!
-
-	familyman
-		name = "The Family Man"
-		icon_state = "thefamilyman"
-		gender = MALE
-
-	mahdrills
-		name = "Drillruru"
-		icon_state = "drillruru"
-		gender = FEMALE
-
-	dandypomp
-		name = "Dandy Pompadour"
-		icon_state = "dandypompadour"
-		gender = MALE
-
-	poofy
-		name = "Poofy"
-		icon_state = "poofy"
-		gender = FEMALE
-
-	crono
-		name = "Chrono"
-		icon_state = "toriyama"
-		gender = MALE
-
-	vegeta
-		name = "Vegeta"
-		icon_state = "toriyama2"
-		gender = MALE
-
-	cia
-		name = "CIA"
-		icon_state = "cia"
-		gender = MALE
-
-	mulder
-		name = "Mulder"
-		icon_state = "mulder"
-		gender = MALE
-
-	scully
-		name = "Scully"
-		icon_state = "scully"
-		gender = FEMALE
-
-	nitori
-		name = "Nitori"
-		icon_state = "nitori"
-		gender = FEMALE
-
-	joestar
-		name = "Joestar"
-		icon_state = "joestar"
-		gender = MALE
-
-	volaju
-		name = "Volaju"
-		icon_state = "volaju"
-
-	longeralt2
-		name = "Long Hair Alt 2"
-		icon_state = "longeralt2"
-
-	shortbangs
-		name = "Short Bangs"
-		icon_state = "shortbangs"
-
-	halfshaved
-		name = "Half-Shaved Emo"
-		icon_state = "halfshaved"
-
-	bun
-		name = "Bun"
-		icon_state = "bun"
-
-	doublebun
-		name = "Double-Bun"
-		icon_state = "doublebun"
+/datum/sprite_accessory/hair/shorthair2
+	name = "Short Hair 2"
+	icon_state = "shorthair2"
 
-	Mia
-		name = "Mia"
-		icon_state = "mia"
-		gender = FEMALE
+//datum/sprite_accessory/hair/shorthair3
```

Comment:
Pull requests should not contain commented code except TODOs and explanation comments. Please remove all commented lines.